fastly/block_fastly_service_v1_logging_scalyr.go

```diff
@@ -0,0 +1,208 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/fastly/go-fastly/fastly"
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var scalyrloggingSchema = &schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			// Required fields
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The unique name of the Scalyr logging endpoint.",
+			},
+
+			"token": {
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
The docs don't say required, but it's required in the [CLI](https://github.com/fastly/cli/blob/0264e0b19e942ad082a25506567a82aa96fe8116/pkg/logging/scalyr/create.go#L43). Confirm that this is required by the backend.